src/test/java/com/google/javascript/gents/singleTests/goog_scope.ts

```diff
@@ -2,14 +2,14 @@ export class Foo {
   static num: number = 8;
   constructor(public n: number) {}
 
-  // TODO: Aggressively export rather than create static methods/fields
-  static foo(): number {
-    return 4;
+  static foo(): string {
+    return 'this should create a static method on Foo, since it is NOT goog.provided';
   }
 
   // TODO: Aggressively export rather than create static methods/fields
-  static qux(): number {
-    return 4;
+  static qux(): string {
+    return 'this should be directly exported, rather than create a static method, ' +
```

Comment:
It isn't though, is it? I don't mind that (I think exporting a static function of a class is a rare corner case), but maybe make it clearer language wise that this is not intended behaviour?